components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -452,95 +461,138 @@ private String getOIDCAuthzEndpoint(Map<String, String> authenticatorProperties)
     protected void processAuthenticationResponse(HttpServletRequest request, HttpServletResponse response,
                                                  AuthenticationContext context) throws AuthenticationFailedException {
 
-        try {
+        // oAuthResponse can be null in some authentication flows. i.e Google One Tap.
+        OAuthClientResponse oAuthResponse = generateOauthResponse(request, context);
+        // TODO : return access token and id token to framework
```

Comment:
Is this under development?

Reply:
Early comment, shown as edited due to try catch block